src/input.rs

```diff
@@ -59,20 +61,27 @@ impl MidiInput {
     ///
     /// Change detection is fired when the ports are refreshed.
     pub fn refresh_ports(&self) {
-        self.sender.send(Message::RefreshPorts).unwrap();
```

Comment:
I don't really see a reason for these messages to be constants. I'd argue that there's not much gained from the deduplication across `MidiInput`/`MidiOutput`, and it makes our code less readable to put the panic message elsewhere.